android/app/src/main/cpp/peer_audio_manager.cpp

```
@@ -447,6 +484,52 @@ void PeerAudioManager::sendAudioToPeer(JNIEnv* env,
+        if (callbackObject_) {
+            callback = env->NewLocalRef(callbackObject_);
+        }
+    }
+    if (!callback) return;
+
+    jclass callbackClass = env->GetObjectClass(callback);
+    if (callbackClass) {
+        jmethodID method = env->GetMethodID(
+            callbackClass, "onTalkingPeersChanged", "([Ljava/lang/String;)V");
+        if (method) {
+            jclass stringClass = env->FindClass("java/lang/String");
```

Clang (14.0.6)

[warning] variable 'stringClass' is not initialized

(cppcoreguidelines-init-variables)